lib/global-styles.php

```diff
@@ -495,13 +495,20 @@ function gutenberg_experimental_global_styles_merge_trees( $core, $theme, $user
 	$result = gutenberg_experimental_global_styles_normalize_schema( array() );
 
 	foreach ( array_keys( $core ) as $block_name ) {
-		foreach ( array( 'presets', 'styles', 'features' ) as $subtree ) {
+		foreach ( array( 'presets', 'features' ) as $subtree ) {
 			$result[ $block_name ][ $subtree ] = array_merge(
 				$core[ $block_name ][ $subtree ],
 				$theme[ $block_name ][ $subtree ],
 				$user[ $block_name ][ $subtree ]
 			);
 		}
+		foreach ( array_keys( $core[ $block_name ]['styles'] ) as $subtree ) {
```

Comment:
Do we expect the "core" styles to always contain all the keys?

Reply:
With the current implementation, yes, as they all are normalized before being merged.